### PR TITLE
Fix vim shell

### PR DESCRIPTION
**Pull request by christopher-haueter**

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -221,3 +221,4 @@ else
   let &t_EI = "\<Esc>]50;CursorShape=0\x7"
 end
 
+set shell=/bin/sh
```
Comment (arktisklada) [inline vimrc:224]: does this need to be the same as the tmux shell?  I believe our iterm/tmux config uses zsh.  Is there a specific error this is resolving?

Comment (christopher-haueter) [inline vimrc:224]: this is resolving the error that running specs from vim while in tmate attempts to use an incorrect ruby version

this is the suggested rvm solution for vim (https://rvm.io/integration/vim)

there's no problem using /bin/sh as our vim shell. we don't use anything zsh-y in our vim config

Comment (amygimma) [inline vimrc:224]: this has been working for me, this solved my ruby versioning problems

**Comment by amygimma** [comment]
:+1:

**Pull request merged**